ctl/secret/secret.go

```diff
@@ -36,6 +36,7 @@ import (
 )
 
 var log = logger.NewLoggerScope("kmeshctl/secret")
+var clientset kube.CLIClient
```

Comment:
Basically i am not a fan of global varible, you can pass the client in the function arguments